tests/e2e/tests/simple/testdata/v1alpha3/servicesToBeInjected.yaml

```diff
@@ -27,6 +27,16 @@ spec:
     - "*"
 ---
 apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: echosrv
+spec:
+  host: 'echosrv.istio-system.svc.cluster.local'
+  trafficPolicy:
+    tls:
+      mode: DISABLE​
```

Comment:
somehow this got a zero width space character after `DISABLE` which is breaking the tests

Reply:
Thanks, for digging into it. I removed the character, still wondering how I managed to add it there.